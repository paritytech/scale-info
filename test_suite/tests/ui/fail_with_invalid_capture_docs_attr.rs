use info::{self as scale_info};
use scale_info::TypeInfo;
use scale::Encode;

#[derive(TypeInfo, Encode)]
#[scale_info(capture_docs = "invalid")]
/// Docs
struct InvalidDocsCapture {
    /// Docs
    a: u8,
}

fn main() {}
