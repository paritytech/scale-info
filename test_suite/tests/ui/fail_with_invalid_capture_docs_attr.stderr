error: Invalid capture_docs value. Expected one of: "default", "always", "never"
 --> tests/ui/fail_with_invalid_capture_docs_attr.rs:6:29
  |
6 | #[scale_info(capture_docs = "invalid")]
  |                             ^^^^^^^^^
